app/infrastructure/database/models.py

```diff
@@ -1,5 +1,113 @@
-from sqlalchemy.orm import DeclarativeBase
```

Comment:
1. use mapped  column instead of column
2. Add typing to fields, e.g id: Mapped[int] = ...
3. remove salt, image, init
4. Add private bool field